Toolset/libraries/revidedeveloperextensionlibrary.livecodescript

```diff
@@ -568,37 +575,46 @@ on revIDEDeveloperExtensionUninstall pPath
 end revIDEDeveloperExtensionUninstall
 
 on revIDEDeveloperExtensionEditScript pFolder
-   local tFile
-   put pFolder & slash & __revIDEDeveloperExtensionFetchModuleInFolder(pFolder) into tFile
-   
+   local tFile, tType
+   __revIDEDeveloperExtensionFetchExtensionSourceInFolder pFolder, tFile, tType
+   if tType is "lcb" then
+      __revIDEDeveloperExtensionEditLCBScript pFolder & slash & tFile
+   else if there is a stack (pFolder & slash & tFile) then
+      edit script of stack (pFolder & slash & tFile)
+   end if
+end revIDEDeveloperExtensionEditScript
+
+private command __revIDEDeveloperExtensionEditLCBScript pFile
    # AL-2015-04-01: [[ Bug 15130 ]] First check to see if there is an existing association
-   launch document tFile
+   launch document pFile
    if the result is not empty then
       # AL-2015-04-01: [[ Bug 15130 ]] Check to see if the text editor preference is set
       local tEditor
       put revIDEGetPreference("LCB_textEditor") into tEditor
       if tEditor is not empty then
-         launch tFile with tEditor
+         launch pFile with tEditor
       end if
```

Comment:
@livecodeali this seems a little odd that the OS pref would take precedence over the local IDE pref

Reply:
hmm, yeah, might be worth filing a bug report.